api/api.go

```diff
@@ -99,6 +100,17 @@ func (siw *ServerInterfaceWrapper) GetDevHelloDataset(w http.ResponseWriter, r *
 		return
 	}
 
+	// ------------- Optional query parameter "geotype" -------------
+	if paramValue := r.URL.Query().Get("geotype"); paramValue != "" {
+
```

Comment:
Wondered if there should be something between curly brackets?

Reply:
That file is generated from the swagger spec. That part could be where constraints in the swagger are enforced. In our case we don't have any constraints.